### PR TITLE
[3.2.1 release]: Add 3.2.1 compatibility to the tables

### PR DESCRIPTION
**Pull request by oleorhagen**
Changelog: None
Signed-off-by: Ole Petter <ole.orhagen@northern.tech>

### CODE REVIEW

**Review by lluiscampos — APPROVED** [review]

File: 02.Overview/14.Compatibility/docs.md [hunk 1]
```diff
@@ -92,6 +92,7 @@ Clarification of the table:
 | Mender client 2.6.x             | community                 | community             | stable             |
 | Mender client 3.0.x             | no                        | community             | stable<sup>3</sup> |
 | Mender client 3.1.x             | no                        | community             | stable<sup>3</sup> |
+| Mender client 3.2.x             | no                        | community             | stable<sup>3</sup> |
```
Comment (lluiscampos) [inline 02.Overview/14.Compatibility/docs.md:96]: @kacf @oleorhagen,

I am wondering if we should we have a compatibility table with `mender-connect` too similar to the one with mender-artifact.

We have to write somewhere that mender-connect 2.0.0 is downprioritized in stable (same as mender-client 3.0.0) and right now there is no good place to put it in the docs.

Comment (oleorhagen) [reply 02.Overview/14.Compatibility/docs.md:96]: Good point!

Comment (oleorhagen) [reply 02.Overview/14.Compatibility/docs.md:96]: I'll have a look at it now.

Comment (kacf) [reply 02.Overview/14.Compatibility/docs.md:96]: Yes, good idea.

File: 02.Overview/14.Compatibility/docs.md [hunk 2]
```diff
@@ -152,3 +154,15 @@ The higher version API contains a mix of old and new API endpoints. For endpoint
 |-------------------------------|--------|--------|
 | Mender client 1.x.x and 2.x.x | yes    | no     |
 | Mender client 3.0.0 and later | yes    | yes    |
+
+
+## Mender client and mender connect
+
+<!--AUTOVERSION: "Mender % / mender-artifact %"/ignore-->
+|     mender-client / mender-connect | mender-connect version |
```
Comment (lluiscampos) [inline 02.Overview/14.Compatibility/docs.md:162]: I think we should use "Mender client" since is the naming we are using in the tables above

**Review by lluiscampos — COMMENTED** [review]
We need to add something like
```
!!! <sup>3</sup> mender-connect 2.0.0 and later are not installed by default in Yocto branches 3.1 (dunfell) and older. To enable this or a later version, please see [the `PREFERRED_VERSION` setting when configuring the Yocto build](../../05.System-updates-Yocto-Project/03.Build-for-demo/docs.md#configuring-the-build).
```

But I am unsure where. The new table relates Mender client and mender-connect but not really the yocto branches :thinking:

**Comment by oleorhagen** [comment]
> We need to add something like
> 
> ```
> !!! <sup>3</sup> mender-connect 2.0.0 and later are not installed by default in Yocto branches 3.1 (dunfell) and older. To enable this or a later version, please see [the `PREFERRED_VERSION` setting when configuring the Yocto build](../../05.System-updates-Yocto-Project/03.Build-for-demo/docs.md#configuring-the-build).
> ```
> 
> But I am unsure where. The new table relates Mender client and mender-connect but not really the yocto branches thinking

I added it as a note here, don't really see a more fitting place.

**Comment by oleorhagen** [comment]
@mender-test-bot 
cherry-pick to:
* 3.2.x
* hosted

**Comment by lluiscampos** [comment]
> > We need to add something like
> > ```
> > !!! <sup>3</sup> mender-connect 2.0.0 and later are not installed by default in Yocto branches 3.1 (dunfell) and older. To enable this or a later version, please see [the `PREFERRED_VERSION` setting when configuring the Yocto build](../../05.System-updates-Yocto-Project/03.Build-for-demo/docs.md#configuring-the-build).
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > But I am unsure where. The new table relates Mender client and mender-connect but not really the yocto branches thinking
> 
> I added it as a note here, don't really see a more fitting place.

I submitted an small amend here:
* https://github.com/mendersoftware/mender-docs/pull/1647

**Pull request merged**